docs/source/using/addons.adoc

```diff
@@ -31,65 +31,60 @@ oc adm policy add-scc-to-group anyuid system:authenticated
+NOTE: The `oc` command is executed as *system:admin*.
 
 openshift::
-If the add-on command starts with _openshift_, you can run the `openshift` binary within
-the container that runs OpenShift. This means that any file parameters or other system-specific
-parameters must match the environment of the container instead of your host.
+If the add-on command starts with `openshift`, you can run the *openshift* binary within the container that runs OpenShift.
+This means that any file parameters or other system-specific parameters must match the environment of the container instead of your host.
 
 docker::
-If the add-on command starts with _docker_, it executes a `docker` command against the Docker daemon
-within the Minishift VM. This is the same daemon on which the single-node OpenShift cluster is
-running as well. This is similar to running `eval $(minishift docker-env)` on your host and
-then executing any `docker` command. See also
-link:../command-ref/minishift_docker-env{outfilesuffix}[`minishift docker-env`].
+If the add-on command starts with `docker`, it executes a `docker` command against the Docker daemon within the Minishift VM.
+This is the same daemon on which the single-node OpenShift cluster is running as well.
+This is similar to running `eval $(minishift docker-env)` on your host and then executing any `docker` command.
+See also xref:../command-ref/minishift_docker-env.adoc#minishift-docker-env[`minishift docker-env`].
 
 echo::
-If the add-on command starts with _echo_, the arguments following the _echo_ command are printed
-to the console. This can be used to provide additional feedback during add-on execution.
+If the add-on command starts with `echo`, the arguments following the `echo` command are printed to the console.
+This can be used to provide additional feedback during add-on execution.
 
 sleep::
-If the add-on command starts with _sleep_, it waits for the specified number of seconds. This can be
-useful in cases where you know that a command such as `oc` might take a few seconds before a certain
-resource can be queried.
+If the add-on command starts with `sleep`, it waits for the specified number of seconds.
+This can be useful in cases where you know that a command such as `oc` might take a few seconds before a certain resource can be queried.
 
 NOTE: Trying to add a command that is not listed will cause an error when the add-on gets parsed.
 
 [[addon-variable-interpolation]]
-=== Variable interpolation
+=== Variable Interpolation
 
 Minishift also allows the use of variables as part of the supported commands.
-Variables have the from _#{<variable-name>}_. <<example-addon-routing-variable>>
-shows how the routing suffix can be interpolated into a _openshift_ command
-to create a new certificate as part of securing the OpenShift registry.
+Variables have the format `#{<variable-name>}`.
 
 [[example-addon-routing-variable]]
 .Example: Usage of the routing suffix variable
+
```

Comment:
You're right, I missed that one. Will pull the text out and rephrase to flow into the example heading.